client/src/app/components/Layout/Layout.jsx

```diff
@@ -131,102 +137,133 @@ export class Layout extends PureComponent {
     saveSettingsData(settings);
   };
 
+  handleCookieAccept = () => this.setState({ isCookieSet: true });
+
   render() {
     const {
       currentUser,
-      intl: { formatMessage }
+      intl: { formatMessage },
+      location: { pathname }
     } = this.props;
-    const { pending, pendingForViewType, viewType } = this.state;
+    const { isCookieSet, pending, pendingForViewType, viewType } = this.state;
+    const isNotPrivacyPolicyPage = !pathname.includes('privacy-policy');
```

Comment:
reverse this please to positive `const isPrivacyPolicyPage `
and negate this later where needed